### PR TITLE
Free up memory allocated by Pools during any PPTC translations at boot time.

### PR DESCRIPTION
**Pull request by LDj3SNuD**
`Ptc.MakeAndSaveTranslations()` no longer uses `Parallel.ForEach()` but `Thread`, to avoid having translation threads with too long a lifetime and too much turnover of the same threads at the end of the job.

Nits.

`ARMeilleure.Common.BitMap` pools are not involved.

```
NES Online:
  7904 (00:25) Translations:
    ~1600 MB: Memory to menu without ResetPools()
    ~800 MB: Memory to menu with ResetPools()
```

### CODE REVIEW

File: ARMeilleure/Common/ThreadStaticPool.cs [hunk 1]
```diff
@@ -33,9 +35,10 @@ private static Stack<ThreadStaticPool<T>> GetPools(int groupId)
         public static void PreparePool(int groupId)
         {
             // Prepare the pool for this thread, ideally using an existing one from the specified group.
+
             if (_instance == null)
             {
-                Stack<ThreadStaticPool<T>> pools = GetPools(groupId);
+                var pools = GetPools(groupId);
```
Comment (riperiperi) [inline ARMeilleure/Common/ThreadStaticPool.cs:41]: Switch to var for a function with no obvious type? I don't think we normally do this. (though `Stack<ThreadStaticPool<T>>` was rather verbose.

**Review by riperiperi — APPROVED** [review]
Looks good, this one was pretty rough on RAM. Just one small question about the use of `var`, otherwise looks good.

File: ARMeilleure/Common/ThreadStaticPool.cs (hunk 1 above)
Comment (LDj3SNuD) [reply ARMeilleure/Common/ThreadStaticPool.cs:41]: It's just a matter of verbosity, I can change if you want.

**Review by Thog — APPROVED** [review]
lgtm!

**Review by gdkchan — APPROVED** [review]

**Pull request merged**